docs/build/configure/_index.md

```diff
@@ -264,6 +264,123 @@ The corresponding raw JSON looks like this:
+      }
+    }
+  ]
+```
+
+{{% /tab %}}
+{{% tab name="JSON Example" %}}
+
+```json {class="line-numbers linkable-line-numbers"}
+{
+  "components": [
+    {
+      "name": "local",
+      "model": "rdk:builtin:pi",
+      "type": "board",
+      "namespace": "rdk",
+      "attributes": {},
+      "depends_on": []
+    },
+    {
+      "name": "my_temp_sensor",
+      "model": "rdk:builtin:bme280",
+      "type": "sensor",
+      "namespace": "rdk",
+      "attributes": {},
+      "depends_on": [],
+      "service_configs": [
+        {
+          "type": "data_manager",
+          "attributes": {
+            "capture_methods": [
+              {
+                "method": "Readings",
+                "additional_params": {},
+                "capture_frequency_hz": 0.017
+              }
+            ]
+          }
+        }
+      ]
+    }
+  ],
+  "webhooks": [
+    {
+      "url": "https://examplecloudfuntionURL-besuretochangethis.net/12345",
+      "event": {
+        "attributes": {
+          "data_types": ["binary", "tabular"]
+        },
+        "type": "part_data_ingested"
+      }
+    }
+  ]
+}
+```
+
+{{% /tab %}}
+{{< /tabs >}}
+
+2. Replace the URL value with the URL of your cloud/lambda function.
+3. Edit the `data_types` list to include only the types of data you want to trigger on.
+4. Configure [data capture](/data/capture/) and [cloud sync](/data/cloud-sync/) for the relevant components.
+   For example, if you want to trigger a webhook on temperature readings, configure data capture and sync on your temperature sensor.
+   Be aware that the component must return the type of data you configured in `data_types`.
+5. Write your lambda function to process the request from `viam-server`.
+   The following example function sends a Slack message with the machine details when it receives a request:
+
+   ```python {class="line-numbers linkable-line-numbers"}
+   import functions_framework
+   import requests
+   import time
+
+   @functions_framework.http
+   def hello_http(request):
+     payload = {
+       "Org-ID": request.headers['org-id'] if 'org-id' in request.headers else 'no value',
+       "Location-ID": request.headers['location-id'] if 'location-id' in request.headers else 'no value',
+       "Part-ID": request.headers['part-id'] if 'part-id' in request.headers else 'no value',
+       "Robot-ID": request.headers['robot-id'] if 'robot-id' in request.headers else 'no value'
+     }
+
+     slack_url = "<paste in your own Slack URL"
+     headers = {}
+
+     response = requests.post(slack_url, json=payload, headers=headers)
+
+     request_json = request.get_json(silent=True)
+     request_args = request.args
+
+     return 'Sent request to {}'.format(slack_url)
+
+   ```
+
+   You may find this tutorial from Ordergroove Engineering on [Posting Slack notifications using Google Cloud Function](https://medium.com/ordergroove-engineering/posting-slack-notifications-using-google-cloud-function-900ed2f1e3ec) helpful.
```

Comment:
I don't think we should assume that the this is a likely use case. It's nice to give an example but I think we can leave it at that
```suggestion
```